modules/virt-understanding-scratch-space.adoc

```diff
@@ -16,12 +16,12 @@ If there is no default storage class defined in the cluster, the storage class u
 
 [NOTE]
 ====
-The CDI requires requesting scratch space with a `file` volume mode, regardless of the PVC backing the origin data volume.
+CDI requires requesting scratch space with a `file` volume mode, regardless of the PVC backing the origin data volume.
 If the origin PVC is backed by `block` volume mode, you must define a storage class capable of provisioning `file` volume mode PVCs.
 ====
 
 [discrete]
 == Manual provisioning
 
-If there are no storage classes, the CDI will use any PVCs in the project that match the size requirements for the image.
-If there are no PVCs that match these requirements, the CDI import pod will remain in a *Pending* state until an appropriate PVC is made available or until a timeout function kills the pod.
+If there are no storage classes, CDI uses any PVCs in the project that match the size requirements for the image.
+If there are no PVCs that match these requirements, CDI import pod remains in a *Pending* state until an appropriate PVC is made available or until a timeout function kills the pod.
```

Comment:
Line 26 is good
Line 27: I think this _should_ have `the` because it's referring to the import pod, not CDI itself